src/confd/yang/infix-system@2023-10-19.yang

```diff
@@ -19,6 +19,14 @@ module infix-system {
   contact      "kernelkit@googlegroups.com";
   description  "Infix augments and deviations to ietf-system.";
 
+  revision 2023-10-19 {
+    description "Add support for timezone-utc-offset
+                Add deviations for timezone-utc-offset:
+                - Metric is set to hours (tzdata compatibility)
+                - Range is -12 .. 14";
```

Comment:
You can reword the first two lines to something like "Change deviation for timezone-utc-offset from unsupported to Etc+/-HOUR".